executor/runtime/docker/docker_linux.go

```diff
@@ -37,15 +39,44 @@ type schedParam struct {
 	schedPriority int32
 }
 
+type serviceOpts struct {
+	humanName       string
+	initCommand     string
+	initCommandArgs string
+	required        bool
+	unitName        string
+}
+
 const (
-	titusInits                 = "/var/lib/titus-inits"
-	atlasSystemdUnit           = "atlas-titus-agent"
-	metadataServiceSystemdUnit = "titus-metadata-proxy"
-	metatronServiceSystemdUnit = "titus-metatron-sync"
-	sshdSystemdUnit            = "titus-sshd"
-	systemServiceStartTimeout  = 90 * time.Second
-	umountNoFollow             = 0x8
-	sysFsCgroup                = "/sys/fs/cgroup"
+	titusInits                = "/var/lib/titus-inits"
+	systemServiceStartTimeout = 90 * time.Second
+	umountNoFollow            = 0x8
+	sysFsCgroup               = "/sys/fs/cgroup"
+)
+
+var (
```

Comment:
In fact, you could make this a list, and then the start unit stuff down below could just walk the list

Reply:
That's a great idea.  Will do.